readthedocs/templates/projects/notifications/deprecated_webhook_endpoint_email.html

```diff
@@ -1,7 +1,8 @@
-<p>Just a heads up, your project {{ project.name }} has configured a <em>DEPRECATED</em> webhook to trigger new builds and should be upgraded. Projects hitting these deprecated webhook will stop building on Jan 1, 2019. Please, update it soon!</p>
+<p>Your project, {{ project.name }}, is currently using GitHub Services to trigger builds on Read the Docs. Effective January 31, 2019, GitHub will no longer process requests using the Services feature, and so Read the Docs will not receive notifications on updates to your repository.</p>
```

Comment:
`{{ project.name }}` is surrounded by two commas, I guess we only need one, right?

Reply:
Grammar time! `{{ project.name}}` would be an appositive, requiring surrounding in commas in this particular use:
http://www.chompchomp.com/terms/appositive.htm

We can avoid commas if we instead write:

`Your {{ project.name }} project is currently using ...`

But this is slightly awkward as `{{ project.name }}` could be interpreted as an adjective given the right project name -- for example, for a project named `docs`:

`Your docs project is currently using...`

And so this is less clear.

Reply:
Thanks for the link!